README.md

```diff
@@ -512,11 +527,18 @@ This section provides examples of interacting with the Envoy External Authorizat
   }
   ```
 
-The `-proto` and `-import-path` flags tell `grpcurl` the relevant proto source file and the folder from which dependencies can be imported respectively. These flags need to be provided as the Envoy External Authorization gRPC server does not support reflection. See this [issue](https://github.com/grpc/grpc-go/issues/1873) for details.
+The `-proto` and `-import-path` flags tell `grpcurl` the relevant proto source file and the folder from which
+dependencies can be imported respectively. These flags need to be provided as the Envoy External Authorization gRPC
+server does not support reflection. See this [issue](https://github.com/grpc/grpc-go/issues/1873) for details.
+
+
+## Examples
 
+See the [examples](./examples) folder for examples of how to deploy the OPA-Envoy plugin with Envoy-based service meshes.
```

Comment:
Let's move this up to the top of the README and list each one of the examples we have w/ bullets. This is an area that others can easily contribute to.

Reply:
Also, `Examples` is not a great section name since there are several other examples on this page. Maybe something like:

## Examples with Envoy-based service meshes

* Istio
* Kuma
* OSM
* ...

Reply:
Done